lib/rubycritic/platforms/linux.rb

```diff
@@ -4,14 +4,10 @@ module Rubycritic
   module Platforms
     class Linux < Base
       BROWSER_MAP = {}.tap do |hash|
-        hash[:chrome] = 'google-chrome'
+        hash[:chrome]   = 'google-chrome'
         hash[:chromium] = 'chromium-browser'
-        hash[:firefox] = 'firefox'
-        hash.default = 'firefox'
-      end
-
-      def can_open?
-        platform == :linux || platform == :unix
+        hash[:firefox]  = 'firefox'
+        hash.default    = 'firefox'
```

Comment:
Of course! How did I not remember that! Haha.
@troessner @superiorlu what do you think? 

Maybe we can even simplify this and not allow the specification of which browser to open on - just launch the default one (99% of the use cases I can imagine).